### PR TITLE
Enable configuration for langnames.json

### PR DESCRIPTION
**Pull request by AnonymousWalker**
The [door43-client](https://github.com/unfoldingWord-dev/node-door43-client) package injects a default URL of `langnames.json` and we made a [fork](https://github.com/AnonymousWalker/node-door43-client/pull/1) to modify and extend the URL field so that the user can change it in App Settings.

This also applies the change to the existing index.sqlite shipped with Writer Desktop - catalog schema

_Note: in the application, the user needs to Update the target language list to see new languages updated from the schema_

### CODE REVIEW

**Comment by AnonymousWalker** [comment]
@PurpleGuitar I'll mark this as Draft until we get the official endpoint for langnames.json

**Review by PurpleGuitar — CHANGES_REQUESTED** [review]
It looks like you updated package/lib/main.js to change the hard-coded langnames.json URL, and then zipped it up into door43-client...tgz.

Please investigate the possibility of making this a configuration setting, configurable via the Settings menu.

**Review by PurpleGuitar — APPROVED** [review]

**Comment by AnonymousWalker** [comment]
updated to production url of langnames.json

File: package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "archiver": "5.3.1",
     "async": "1.4.2",
     "chmodr": "1.0.2",
-    "door43-client": "^0.9.6",
+    "door43-client-fork": "file:src/js/lib/external/door43-client-fork-0.9.8.tgz",
```
Comment (mXaln) [inline package.json:28]: What do you think? Could we make a fork and create this package on npmjs.com, instead of loading this from file?

**Review by mXaln — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (AnonymousWalker) [reply package.json]: thanks, that sounds great

**Review by mXaln — APPROVED** [review]

**Pull request merged**